sites/lab/package.json

```diff
@@ -4,8 +4,10 @@
   "private": true,
   "scripts": {
     "dev": "node --experimental-json-modules ../../node_modules/.bin/next dev -p 8000",
+    "dev-debug": "node --inspect --experimental-json-modules ../../node_modules/.bin/next dev -p 8000",
```

Comment:
There's got to be a better way to inspect the lab, one that doesn't require a whole new command…unfortunately I was having problems with `NODE_OPTIONS='--inspect' yarn lab`.